packages/melos/lib/src/commands/bootstrap.dart

```diff
@@ -171,10 +171,29 @@ mixin _BootstrapMixin on _CleanMixin {
     Package package, {
     bool inTemporaryProject = false,
   }) async {
-    final pubGetArgs = ['pub', 'get'];
-    final execArgs = package.isFlutterPackage
-        ? ['flutter', ...pubGetArgs]
-        : [if (utils.isPubSubcommand()) 'dart', ...pubGetArgs];
+    late final List<String> execArgs;
```

Comment:
Why not just this?
```
final execArgs = [
    if (package.isFlutterApp)
        '${workspace.sdkPath}flutter'
    else if (utils.isPubSubcommand(sdkPath: workspace.sdkPath))
        '${workspace.sdkPath}dart',
    'pub',
    'get',
]
```

Of if `workspac.sdkPath` is nullable:
```
final execArgs = [
    if (package.isFlutterApp)
        '${workspace.sdkPath ?? ''}flutter'
    else if (utils.isPubSubcommand(sdkPath: workspace.sdkPath))
        '${workspace.sdkPath ?? ''}dart',
    'pub',
    'get',
]
```

Reply:
Simplified in ca2d3a8a9ed5fec764c3667fad0702c818593a1e (includes tests)